File: Overture.Core/Domain/Entities/Business.cs
using System;
using System.Collections.Generic;
using System.Text;
using Overture.Core.Domain.ValueObjects;

namespace Overture.Core.Domain.Entities
{
	public class Business : IEntity
	{
		public Guid Id { get; set; }
		public string UserId { get; set; }
		public string Name { get; set; }
		public string Owner { get; set; }
		public string TagLine { get; set; }
		public string Description { get; set; }  
		public decimal MinumumRate { get; set; }
		public bool IsTrading { get; set; }
		public Address Address { get; set; }
		public IEnumerable<BusinessService> BusinessServices { get; set; }
		public IEnumerable<ServiceArea> ServiceAreas { get; set; }

	}
}

File: Overture.Web/clientapp/src/app/_models/business.ts
import { BusinessRatings } from './business-ratings';
import { StoredFile } from '@app/_models/stored-file';
import { Address } from './address';
import { ServiceArea } from './service-area';
import { BusinessService } from './business-service';
import { ContactMethod } from './contact-method';
export class Business {
  id: string;
  name: string;
  altReference:string;
  owner: string;
  tagline: string;
  description: string;
  isTrading: boolean;
  picture: StoredFile;
  address: Address;
  serviceAreas: ServiceArea[];
  businessServices: BusinessService[];
  contactMethods: ContactMethod[];
  storedFiles:StoredFile[];
  ratings:BusinessRatings;
  constructor(){
    this.ratings = new BusinessRatings();
  }
}
